rakelib/form526.rake

```diff
@@ -96,22 +96,19 @@ namespace :form526 do
       end
     end
 
-    def clean_message(msg)
-      if msg[1].present?
-        # strip the GUID from BGS errors for grouping purposes
-        "#{msg[0]}: #{msg[1].gsub(/GUID.*/, '')}"
-      else
-        msg[0]
+    def message_string(msg)
+      return nil if msg.dig('severity') == 'WARN'
+
+      message = msg.dig('key')&.gsub(/\[(\d*)\]|\\/, '')
```

Comment:
asked Anna why strip out backslashes too, she linked me to:
https://github.com/department-of-veterans-affairs/vets-api/pull/4742